stdlib/LinearAlgebra/src/triangular.jl

```diff
@@ -424,8 +424,8 @@ function scale!(A::LowerTriangular, B::Union{LowerTriangular,UnitLowerTriangular
     end
     return A
 end
-scale!(A::Union{UpperTriangular,LowerTriangular}, c::Number) = scale!(A,A,c)
-scale!(c::Number, A::Union{UpperTriangular,LowerTriangular}) = scale!(A,c)
+mul1!(A::Union{UpperTriangular,LowerTriangular}, c::Number) = mul!(A, A, c)
+mul2!(c::Number, A::Union{UpperTriangular,LowerTriangular}) = mul1!(A', c')'
```

Comment:
One too many `'`.

Reply:
I think this one is correct: `c*A = (c*A)'' = ((c*A)')' = (A'*c')'`

Reply:
Oh, I confused myself and thought the line ended with the char `')'`

Reply:
I'm confused — why can't you simply define `mul2!(c::Number, A) = mul1(A, c)`?

Reply:
Maybe we should have
```jl
mul2!(c::Union{Real,Complex}, A::Union{UpperTriangular,LowerTriangular}) = mul1!(A, c)?
```
if it makes a performance difference in the common case.

Reply:
This one was hitting the generic fallback so it ended up being about twice as slow. I realized a better version would be to call `mul!(A, c, A)` and define that method. It also made me fix `mul!(A, A, c)` since it actually computed `c*A`. Eventually, we should fix the `Adjoint`/`Transpose` version so we avoid traversing the zero parts but that is an optimization for a separate PR.